src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -93,6 +94,13 @@
  * <p> Implementations of this interface are immutable and safe for use by
  * multiple concurrent threads.
  *
+ * @apiNote
+ * On some file systems a {@code Path} may have an internal representation
+ * which cannot be converted to a {@code String} without losing information.
+ * Otherwise stated, the internal representation might not be recoverable
+ * from the derived path string. This applies to the {@code Path} as a whole
+ * as well as to its components.
```

Comment:
This comment doesn't seem specific to the enhancement proposed.
And its a bit vague, if/when included it might benefit from an example or mentioning the character set encoding of file names, if that's the context.

Reply:
I am wondering now whether this note would just add unnecessary confusion.

Reply:
Removed confusing, value API note in 198ab9a8ca1c05514abec932cd55d0b8597121d8.